src/theme.js

```diff
@@ -146,13 +146,10 @@ function ThemeNav () {
             if (link.length > 0) {
                 $('.wy-menu-vertical .current').removeClass('current');
                 link.addClass('current');
-                link.closest('li.toctree-l1').addClass('current');
                 link.closest('li.toctree-l1').parent().addClass('current');
-                link.closest('li.toctree-l1').addClass('current');
-                link.closest('li.toctree-l2').addClass('current');
-                link.closest('li.toctree-l3').addClass('current');
-                link.closest('li.toctree-l4').addClass('current');
-                link.closest('li.toctree-l5').addClass('current');
+                for (let i = 1; i <= 10; i++) {
+                    link.closest(`li.toctree-l${i}`).addClass('current');
```

Comment:
I'm not sure about browser support here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#browser_compatibility